engines/decidim-debates/app/models/decidim/debates/debate.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+module Decidim
+  module Debates
+    # The data store for a Debate in the Decidim::Debates component. It stores a
+    # title, description and any other useful information to render a custom
+    # debate.
+    class Debate < Debates::ApplicationRecord
+      include Decidim::Authorable
+      belongs_to :feature, foreign_key: "decidim_feature_id", class_name: Decidim::Feature
+      belongs_to :scope, foreign_key: "decidim_scope_id", class_name: Decidim::Scope
+      belongs_to :category, foreign_key: "decidim_category_id", class_name: Decidim::Category
+      has_one :organization, through: :feature
+
+      validate :category_belongs_to_organization
+      validate :author_belongs_to_organization
+      validates :title, presence: true
+
+      def author_name
```

Comment:
I'd remove the author name from here & the views. Debates are only created by admins so it makes no sense.